packages/plugin-apex/src/commands/force/apex/test/run.ts

```diff
@@ -121,6 +121,14 @@ export default class Run extends SfdxCommand {
       this.ux.warn(messages.getMessage('warningMessage'));
     }
 
+    // W-9346875.  When the wait flag is present, display
```

Comment:
My change below (lines 126-130) seems "not obvious", so I added a comment as to why the code is here, and referenced the ticket ID.  If this is not the team's convention and want it removed, or if anyone would like the wording changed, just let me know.

Reply:
In general we prefer to add "why" comments instead of "what" or "how" when things are not obvious. We have been trying to keep them limited because comments rot really quick when there is constant change happening in the code base :)

Reply:
👍